src/test/java/com/rultor/agents/github/ReportsTest.java

```diff
@@ -56,9 +68,52 @@
     public void reportsRequestResult() throws Exception {
         final Repo repo = new MkGithub().randomRepo();
         final Issue issue = repo.issues().create("", "");
+        final Talk talk = ReportsTest.example(repo, issue);
+        final Agent agent = new Reports(repo.github());
+        agent.execute(talk);
+        MatcherAssert.assertThat(
+            talk.read(),
+            XhtmlMatchers.hasXPath(XPATH)
+        );
+    }
+
+    /**
+     * Reports can report a result of a request, when stop command fails.
+     * @throws Exception In case of error.
+     */
+    @Test
+    public void reportsRequestResultWhenStopFails() throws Exception {
```

Comment:
@mkordas In line: 

`final Issue issue = repo.issues().create("Bug", "stop it please");`

I added comment with `stop` command. So `Reports` creates report on command that was tried to `stop`.